apps/page/option/banned_word.ts

```diff
@@ -64,6 +66,36 @@ export default class BannedWords {
     });
   }
 
+  private assertButtonElement(element: HTMLElement | null): HTMLButtonElement {
```

Comment:
This approach is overly verbose. Whether it's null or defined as a different element, it's a programming error and the same error should suffice. Just adding a check like `if (!(anchor instanceof HTMLAnchorElement)) {` in the constructor, as written in #931, should be enough. Please don't define a function just for that.

Reply:
In this case we are getting element from `getElementById` method is having type of HTMLElement and anyhow we need to do typecasting using as to convert it into `HTMLButtonElement` or required element. So i think we need to revert the changes just add changes like this to avoid error`if (!(button instanceof HTMLButtonElement)) {`.

Reply:
OK, I understand. 
To put it into an instance variable, it needs to be non-null at this point.
Your approach is correct, and the function can also be reused commonly.
I found a redundant "as" casting. Please fix just that.